src/components/link/_link.scss

```diff
@@ -42,3 +43,55 @@
     }
   }
 }
+
+@mixin link--x {
+  .#{$prefix}--link {
+    @include reset;
+    @include font-family;
+    @include font-smoothing;
+
+    color: $ibm-colors__blue--60;
+    font-weight: 400;
+
+    // There is currently no way to set the thinkness of an underline so we'll rely on the box shadow property to achieve this
+    // effect.
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+      color: $ibm-colors__blue--60;
+      box-shadow: 0 1px currentColor;
+    }
+
+    &:active {
+      color: $ibm-colors--gray--100;
+      box-shadow: 0 1px currentColor;
+    }
+
+    &:focus {
+      outline: none;
+      box-shadow: 0 3px currentColor;
+    }
+
+    &[aria-disabled='true'][tabindex='-1'] {
+      // pointer-events: none;
+      color: $ibm-colors__gray--30;
```

Comment:
I added a token to the text input PR `$text-disabled` you could use

Reply:
Was going to wait until you merged your `text-input` PR to change the token, didn't want to override what you have.